lib/spack/spack/cmd/external.py

```diff
@@ -53,6 +53,12 @@ def setup_parser(subparser):
     find_parser.add_argument(
         "--all", action="store_true", help="search for all packages that Spack knows about"
     )
+    find_parser.add_argument(
+        "--debug-find", action="store_true", help="write all candidate paths seached to stdout"
+    )
```

Comment:
Looks like this also includes everything from https://github.com/spack/spack/pull/41642 - was that intended?

Reply:
For debugging it was useful. My intention was to drop those changes (they're one commit) once #41642 lands.